runtime/vm/LayoutFFITypeHelpers.cpp

```diff
@@ -49,8 +49,9 @@ freeAllStructFFITypes(J9VMThread *currentThread, void *cifNode)
 	}
 }
 
-ffi_type*
-LayoutFFITypeHelpers::getArrayFFIType(char **layout, UDATA nElements) {
+ffi_type *
+LayoutFFITypeHelpers::getArrayFFIType(char **layout, UDATA nElements)
+{
 	ffi_type *typeFFI = NULL;
 	ffi_type *elementType = NULL;
 	PORT_ACCESS_FROM_JAVAVM(_vm);
```

Comment:
Please correct the sample `int_struct` below (use legal C syntax):
```c
	/* A struct starts with "#" plus the count of struct elements,
	 * followed by the types of the struct elements.
	 * e.g. the preprocessed layout of
	 *     struct int_struct {
	 *         int elem1;
	 *         int elem2;
	 *     };
	 * is "#2[II]"
	 */
```

Reply:
Fixed as suggested above.